contrib/index.rst

```diff
@@ -1,28 +1,116 @@
+.. _c_root:
+
 ==================================
 Python Contributor's Guide (draft)
 ==================================
 
+.. raw:: html
+
+    <script>
+    document.addEventListener('DOMContentLoaded', function() {
+      activateTab(getOS());
+    });
+    </script>
+    <style>
+      table.docutils td { vertical-align: top; }
+    </style>
+
+
 .. important::
 
    |draft|
 
    |purpose|
 
 
-This guide is a comprehensive resource for contributing to Python.
+[Open question: how to divide content between this Introduction and the
+:ref:`introduction <c_intro>`?]
+
+This guide is a comprehensive resource for :ref:`contributing <contributing>`
+to Python_ -- for both new and experienced contributors. It is :ref:`maintained
+<devguide>` by the same community that maintains Python.  We welcome your
+contributions to Python!
+
+We encourage everyone to contribute to Python. This guide should have
+everything you need to get started and be productive.  If you still have
+questions after reviewing the material in this guide, the `Core Python
+Mentorship`_ group is available to help you through the process.
+
+There are a number of ways to contribute including code, documentation, and
+triaging issues.  We've organized this guide to provide specifics based on the
+type of activity you'll be engaged in.
+
+
+Using this guide
+================
+
+We recommend reading this guide as needed. You can stop where you feel
+comfortable and begin contributing immediately without reading and
+understanding everything.  If you do choose to skip around this guide, be aware
+that it is written assuming preceding sections have been read so you may need
+to backtrack to fill in missing concepts and terminology.
 
-.. note::
-   This is a draft of a new organization for the devguide, turning it into a
-   Contributor's Guide.
+No matter what kind of contribution you'll be making, you should start with
+these common sections:
 
-[Much of the devguide home page will be here.  The Quick Start will be updated
-to distinguish code vs documentation contributions, and moved to those
-sections.  Other changes will be made to explain the different types of
-contribution and how to navigate the guide for your own type.]
+* :ref:`c_intro`
+* :ref:`c_project`
+* :ref:`c_gettingstarted`
 
+Then choose a path based on your type of activity:
+
+*[The original table on the devguide home had a fourth column for Core
+Developers. That made the table wider and more confusing. I don't think core
+team members need a quick intro path since they will have been through the
+devguide before.]*
+
+*[I haven't adjusted the links in the table yet other than to add a link to the
+major section at the top of each column.]*
+
+.. list-table::
```

Comment:
I'm trying to think which is better, lists inside a table, or smaller cells, from an accessibility point of view, and I'm not sure :)

The new table of lists might be better, because there's no relation between the old cells that happen to be next to each other; that is, the old way was just for presentation and not structure. So I think the list might be better?

Reply:
Other people I've talked with definitely felt like the one-link-per-cell style of the current home page table is misleading because the rows have no meaning.